### ISSUE TITLE
508-defect-2 [FOCUS]: Back to top button's focus halo MUST be visible on Safari

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25001]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a keyboard user, I want to see a visible focus halo so I know an element is interactive and accessible to me.

## Details

The back to top button can receive focus, but does not have a focus halo on Safari.

## Acceptance Criteria

- [ ] The back to top button has a visible focus halo

## Steps to Recreate

On a mac, open up Safari. Tab to the back to top button and confirm its focus isn't visible.

## WCAG or Vendor Guidance (optional)

https://www.wuhcag.com/focus-visible/

## Screenshots or Trace Logs

https://user-images.githubusercontent.com/14154792/118993429-82093e00-b953-11eb-8036-091df4781d4e.mov

## Note
There also appears to be a noticeable lag time on Chrome. I'm unsure as to what's causing this.

**Comment by mmiddaugh** [comment]
Accessibility feedback noted in January in #18399

**Comment by kelsonic** [comment]
https://user-images.githubusercontent.com/12773166/127377630-8c7f44bd-5a5e-41eb-a044-7157df445cd4.mov

Hmm, this might be a site-wide issue actually since the CSS selector `button:not([disabled]):focus` is not applying the outline in Safari for the button 🤔 😬

**Comment by kelsonic** [comment]
Ah ha! I got it -- safari does not support transition animations for outline on buttons, removing that custom CSS resolves it 🙂

**Comment by kelsonic** [comment]
PRs up:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/649
https://github.com/department-of-veterans-affairs/vets-website/pull/18065

**Comment by joshkimux** [comment]
Huge fix. Thanks @kelsonic !

**Comment by kelsonic** [comment]
Ticket currently blocked 🙂 https://github.com/department-of-veterans-affairs/vets-website/pull/18065#pullrequestreview-718594875

**Comment by Rleahy22** [comment]
This looks fixe

https://user-images.githubusercontent.com/3144003/134697727-7b9e4bb4-4a98-4b94-8a19-51ff694829d2.mov

d

**Comment by joshkimux** [comment]
Thanks @Rleahy22 ! Closing this one out
